DSCResources/MSFT_xDefaultGatewayAddress/MSFT_xDefaultGatewayAddress.psm1

```diff
@@ -187,77 +212,92 @@ function Test-Properties
             # Is a default gateway address set?
             if ($defaultRoutes)
             {
-                Write-Verbose -Message "Default gateway exists but it should not."
+                Write-Verbose -Message ( @("$($MyInvocation.MyCommand): "
+                    'Default gateway exists but it should not.'
+                    ) -join '' )
                 $requiresChanges = $true
             }
             else
             {
-                Write-Verbose -Message "Default Gateway does not exist which is correct."
+                Write-Verbose -Message ( @("$($MyInvocation.MyCommand): "
+                    'Default Gateway does not exist which is correct.'
+                    ) -join '' )
             }
         }
+    }
+    catch
```

Comment:
Same comment here that I said about xDNSServerAddress is this really needed given Test-ResourceProperty is supposed to throw?